src/workspace/mod.rs

```diff
@@ -162,7 +162,7 @@ impl Workspace {
         F: FnOnce(&ManifestPath) -> Result<()>,
     {
         let tmp_dir = tempfile::Builder::new()
-            .prefix(".cargo-contract_")
+            .prefix("cargo-contract_")
             .tempdir()?;
```

Comment:
How important is it that this name does not conflict with any other name? Because if it is important we should maybe uglify it a bit more.

Reply:
It's a prefix to a unique generated name e.g. `/tmp/cargo_contract_1afbe3` (I made this up but something like that)